conf/webapp.env

```diff
@@ -4,3 +4,4 @@ WEBSOCKET_HOST=vmaas_websocket
 MAX_VMAAS_SERVERS=1
 LOGGING_LEVEL=INFO
 LOGGING_TYPE=DEVEL
+GZIP_RESPONSE_ENABLE=on
```

Comment:
in code is default value "FALSE", there is "on", please make both consistent "TRUE"/"FALSE" or "on"/"off"

Reply:
ok, also added level param to .env file